stable/kubexray/override-values.yaml

```diff
@@ -0,0 +1,41 @@
+# Override values for kubexray.
```

Comment:
@rimusz Did you add this file with a purpose or it by mistake?

Reply:
by mistake, I removed it